examples/kraken-web-proof/vlayer/prove.ts

```diff
@@ -29,9 +30,14 @@ const vlayer = createVlayerClient({
 
 async function generateWebProof() {
   console.log("⏳ Generating web proof...");
-  const result =
-    await Bun.$`vlayer web-proof-fetch --notary ${notaryUrl} --url ${URL_TO_PROVE}`;
-  return result.stdout.toString();
+  const { stdout } = await runProcess("vlayer", [
+    "web-proof-fetch",
+    "--notary",
+    String(notaryUrl),
+    "--url",
+    URL_TO_PROVE,
+  ]);
+  return stdout;
```

Comment:
Still not ideal, but better than the previous solution. Can we also make sure to capture stderr and report its contents to the user in case of an error?

Reply:
Thanks for explaining!